### ISSUE TITLE
cannot `cargo build --release`

### ISSUE THREAD

**Issue by kyoheiu** [lapce/lapce#465]
Environment: 
rustc  ver: 1.60.0
rustup ver: 1.24.3
cargo  ver: 1.60.0
cargo crates: 24
os: Arch Linux
kernel: 5.17.5-arch1-1

```
git clone https://github.com/lapce/lapce.git
z lapce
cargo build --release
    Updating git repository `https://github.com/lapce/glutin`
error: failed to get `glutin` as a dependency of package `druid v0.7.0 (https://github.com/lapce/druid?branch=opengl#43535989)`
    ... which satisfies git dependency `druid` (locked to 0.7.0) of package `lapce-ui v0.0.12 (/home/kyohei/github/lapce/lapce-ui)`

Caused by:
  failed to load source for dependency `glutin`

Caused by:
  Unable to update https://github.com/lapce/glutin?branch=new-keyboard-all#7c313d91

Caused by:
  object not found - no match for id (7c313d91584492961b9efab0d611e35b977c0fea); class=Odb (9); code=NotFound (-3)
```

Seems cargo.lock in lapce/glutin is broken?

**Comment by bugadani** [comment]
*Should* be fixed by a46fb40 but feel free to reopen this if not.

**Comment by kyoheiu** [comment]
Thank you, it's fixed.
